prow/jobs/generated/knative-extensions/kn-plugin-event-main.gen.yaml

```diff
@@ -125,10 +125,12 @@ periodics:
       - bash
       - -c
       - |
-        mkdir -p /root/.kube
-        cat /opt/cluster/ci-script > connect.sh
-        chmod +x connect.sh
-        ./connect.sh ${CI_JOB}
+        cat /opt/cluster/vm-script > /tmp/vm.sh
```

Comment:
Plesase simplify this bash as @cardil already pointed out across all these files.